.github/workflows/tests.yml

```diff
@@ -0,0 +1,45 @@
+name: "Tests"
```

Comment:
Is it worth to create a workflow just for rubocop linting?
Alternatively, we could name this workflow `name: "Lint and test" 

FF2CC

Reply:
For something simple like this gem I don't think the overhead of time and cost is worth it.